pkg/options/options.go

```diff
@@ -209,6 +214,10 @@ func init() {
 		&MiniEnv, "mini",
 		false, "Signal the operator that it is running in a low resource environment",
 	)
+	FlagSet.BoolVar(
+		&DevEnv, "dev",
+		false, "Set sufficient resoureces for dev env (Arm Macs)",
```

Comment:
Please change `Arm` to `ARM`

Reply:
decided to drop the reference to arm altogether